ocs_ci/ocs/cluster.py

```diff
@@ -508,7 +508,7 @@ def run(self):
             health_status = self.ceph_cluster.get_ceph_health(detail=True)
             if "HEALTH_ERROR" in health_status:
                 self.ceph_cluster.health_error_status = (
-                    self.ceph_cluster.ceph_status()
+                    self.ceph_cluster.get_ceph_status()
```

Comment:
https://github.com/red-hat-storage/ocs-ci/pull/1167/files#diff-e2c2dcfab854db263b9a45a42d6f4b55R454 this is correct name. Fixing it.